### PR TITLE
Snapshot config can be set in schema / properties YAML file starting v1.9

### LINKED ISSUE #5627: [Core] Snapshot config can be set in schema / properties YAML file
At a minimum, update body to include a link to the page on docs.getdbt.com requiring updates and what part(s) of the page you would like to see updated.

Issue: https://github.com/dbt-labs/dbt-core/issues/4000
Pull request: https://github.com/dbt-labs/dbt-core/pull/10275

On page https://docs.getdbt.com/reference/snapshot-configs, there is the line "The following resource-specific configurations are only available to Snapshots:" followed by a line with three tabs: "Project file  Property file. Config block".  If you click "Property file" it says: "Note: Required snapshot properties will not work when defined in config YAML blocks. We recommend that you define these in dbt_project.yml or a config() block within the snapshot .sql file."  This is no longer true. 

There should be a block displaying the config:
```
snapshots:
  - name: my_snapshot
    config:
      schema: specific_schema
      unique_key: specific_id
      strategy: timestamp
      updated_at: updated_at
```

Note: the use of schema instead of "target_schema" is a new thing in versionless and 1.9. I'm not sure how you handle that.

### Example

In Versionless and v1.9 it is possible to specify a snapshot in a SQL file and add configurations in a YAML file like below:

`snapshots/orders_snapshot_file.sql`

```sql
{% snapshot orders_snapshot %}
select * from {{ ref('orders') }}
{% endsnapshot %}
```

`snapshots/_snapshots.yml` or `models/schema.yml` or ...

```yaml
snapshots:
  - name: orders_snapshot
    config:
      target_schema: test
      strategy: check
      unique_key: id
      check_cols: ['status']
```

### PR DESCRIPTION
**Pull request by dbeatty10**
[Preview](https://docs-getdbt-com-git-dbeatty10-patch-2-dbt-labs.vercel.app/reference/snapshot-configs)

resolves: https://github.com/dbt-labs/docs.getdbt.com/issues/5627

## What are you changing in this pull request and why?

Snapshot config can be set in schema / properties YAML file starting v1.9

## 🎩 

v1.8 and below:

image

Versionless and v1.9 and above:

image

## Checklist
- [x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] Add a note to the prerelease version [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/docs/dbt-versions/core-upgrade)

### CODE REVIEW

File: website/docs/reference/snapshot-configs.md [hunk 1]
```diff
@@ -80,7 +80,31 @@ snapshots:
...
+
+**Note:** Required snapshot properties _will not_ work when defined in `config` YAML blocks. We recommend that you define these in `dbt_project.yml` or a `config()` block within the snapshot `.sql` file or upgrade to v1.9.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+<File name='snapshots/schema.yml'>
+
+```yml
```
Comment (mirnawong1) [inline website/docs/reference/snapshot-configs.md:93]: hey @dbeatty10 i actually added a similar spec [here](https://github.com/dbt-labs/docs.getdbt.com/pull/6190/files#diff-0a2da01bb806c897af6340caae18b30748370521c84dd445e8cb35fdeb35aef1R103) as well!

**Review by mirnawong1 — APPROVED** [review]
lgtm @dbeatty10 and added a release note for this!

File: website/docs/docs/dbt-versions/release-notes.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ Release notes are grouped by month for both multi-tenant and virtual private clo
 
 ## October 2024
 
+- **Enhancement**: In dbt Cloud Versionless, you can now specify snapshot configurations in YAML files, including `schema.yml` or `dbt_project.yml`, for snapshots defined in SQL files. This update resolves the previous limitation that required snapshot properties to be defined exclusively in the SQL file. This enhancement will soon be released in the upcoming dbt Core v1.9 release
```
Comment (mirnawong1) [inline website/docs/docs/dbt-versions/release-notes.md]: @dbeatty10 added a release note entry for this - can yo triple check

File: website/docs/reference/snapshot-configs.md [hunk 3]
```diff
@@ -80,7 +80,31 @@ snapshots:
 
 <TabItem value="property-yaml">
 
-**Note:** Required snapshot properties _will not_ work when defined in `config` YAML blocks. We recommend that you define these in `dbt_project.yml` or a `config()` block within the snapshot `.sql` file.
+<VersionBlock lastVersion="1.8">
+
+**Note:** Required snapshot properties _will not_ work when defined in `config` YAML blocks. We recommend that you define these in `dbt_project.yml` or a `config()` block within the snapshot `.sql` file or upgrade to v1.9.
```
Comment (dbeatty10) [inline website/docs/reference/snapshot-configs.md]:
```suggestion
**Note:** Required snapshot properties _will not_ work when only defined in `config` YAML blocks. We recommend that you define these in `dbt_project.yml` or a `config()` block within the snapshot `.sql` file or upgrade to v1.9.
```

File: website/docs/docs/dbt-versions/release-notes.md (hunk 2 above)
Comment (dbeatty10) [reply website/docs/docs/dbt-versions/release-notes.md]:
```suggestion
- **Enhancement**: In dbt Cloud Versionless, snapshots defined in SQL files can now use `config` defined in `schema.yml` YAML files. This update resolves the previous limitation that required snapshot properties to be defined exclusively in `dbt_project.yml` and/or a `config()` block within the SQL file. This enhancement will be included in the upcoming dbt Core v1.9 release.
```

File: website/docs/reference/snapshot-configs.md [hunk 4]
```diff
@@ -80,7 +80,31 @@ snapshots:
...
+
+snapshots:
+  - name: <string>
+    config:
+      database: <string>
+      schema: <string>
+      unique_key: <column_name_or_expression>
+      strategy: timestamp | check
+      updated_at: <column_name>
+      check_cols: [<column_name>] | all
```
Comment (dbeatty10) [inline website/docs/reference/snapshot-configs.md]: Update newlines to match the `dbt_project.yml` tab:

```suggestion
snapshots:
  - name: <string>
    config:
      database: <string>
      schema: <string>
      unique_key: <column_name_or_expression>
      strategy: timestamp | check
      updated_at: <column_name>
      check_cols: [<column_name>] | all

```

**Pull request merged**